aries-site/src/examples/components/avatar/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
This and the AvatarExample file seem swapped.

Reply:
Thank you.